docs/contribute/code.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Code contributions'
+---
+
+If you're willing to fix a bug or add a new feature to [Storybook](https://github.com/storybookjs/storybook), the first step is to set up your development environment. This page outlines how to get set up to contribute to Storybook's code. For instructions on working with documentation, check the [documentation contribution](./documentation-updates.md) page. To contribute  to the code snippets, check the [snippets contribution](./new-snippets.md) page.
+
+## Initial setup
+
+The first thing you need to do is [fork](https://docs.github.com/en/github/getting-started-with-github/quickstart/fork-a-repo) the Storybook repository. Afterward, you need to clone and build your fork locally. Run the following commands:
+
+```shell
+git clone https://github.com/your-username/storybook.git
+cd storybook
+yarn
+yarn bootstrap --core
+```
+
+<div class="aside">
+💡 <strong>Note:</strong> If you're using Yarn 2, you can skip running <code>yarn</code> to install the dependencies.
+</div>
```

Comment:
```suggestion
```

The repo **is** using Yarn 2 and you always need to run an install, you can't run a script without installing first as the dependencies the script _might_ need access to isn't available without it so Yarn throws.

Reply:
@merceyz noted. Thank you for the feedback. I'll push a commit in a bit to address this.

Reply:
Just pushed a commit to address this comment. Marking it as closed.